cdragontoolbox/export.py

```diff
@@ -659,3 +662,31 @@ def convert(self, fin, output, path):
                 name = os.path.join(obj_output_path, entry["name"] + ".obj")
                 with open(name, "w") as f:
                     f.write(sknfile.to_obj(entry))
+
+class RstConverter(FileConverter):
+    def __init__(self, regex):
+        self.regex = regex
+        self.hashes = HashFile(os.path.join(os.path.dirname(__file__), "hashes.rst.txt"), hash_size=16).load()
+
+    def is_handled(self, path):
+        return self.regex.search(path) is not None
+
+    def converted_paths(self, path):
+        yield path + '.json'
+
+    def convert(self, fin, output, path):
+        output_path = os.path.join(output, path + '.json')
+        rstfile = RstFile(fin)
+        bit_mask = (1 << rstfile.hash_bits) - 1
+        hashes = {hash & bit_mask: value for hash, value in self.hashes.items()}
+        rst_json = {}
+        if rstfile.font_config:
+            rst_json["__fontconfig"] = rstfile.font_config
+        for key, value in rstfile.entries.items():
+            if key in hashes:
+                key = hashes[key]
+            rst_json[key] = value
+        rst_json["__version"] = rstfile.version
```

Comment:
Since this is a newly defined JSON format we may avoid the `__special` entries and just put the actual map into its dedicated entry.
Somethink like this
```
{
  "entries": {"some_key": "Some translation", ...},
  ...
}
```

Also, I wonder if we should include `fontconfig` and `version`. `fontconfig` is some obscure blob, not really parsed. And `version` is only relevant for parsing AFAIK, so we don't really need it at this point.

We could decide to add them in the future if they prove to be useful; but it could be wiser to not commit to much on the format for now.

Reply:
I specifically added `version` because of the hash. For version 4, we have 39 bits, and for version 3 and lower 40 bits.
It's impossible to know whether a hash has the 40s bit set to 0 or is a 39-bit length hash just by looking at the number, which is what I'm mainly targeting with this thing.
Including fontconfig is also mainly for not losing information that was originally in the binary file.

Reply:
Ok for `version`. For `fontconfig` we can keep it as it is... and maybe rework it one day. But since I doubt anyone will be using it anyway, that would be safer to just not put it. Anyone is interested can parse the original file.

Reply:
I guess I'll just the fontconfig then. It only exists for old files and seems to contain outdated information (always `[FontConfig "English"]` for all languages) anyway.